source/s3_meta_request.c

```diff
@@ -439,21 +440,11 @@ void aws_s3_meta_request_send_next_request(
         AWS_LOGF_ERROR(
             AWS_LS_S3_META_REQUEST, "id=%p Could not prepare request %p", (void *)meta_request, (void *)request);
 
-        int error = aws_last_error();
+        aws_s3_meta_request_handle_error(meta_request, request, aws_last_error());
 
-        if (error == AWS_ERROR_S3_NO_PART_BUFFER) {
-            AWS_LOGF_INFO(
-                AWS_LS_S3_META_REQUEST,
-                "id=%p Pushing request into retry queue due to lack of part buffer availability.",
-                (void *)meta_request);
+        aws_s3_request_release(request);
 
-            aws_s3_meta_request_queue_retry(meta_request, request);
-            aws_s3_request_release(request);
-
-            goto call_finished_callback;
-        } else {
-            goto error_finish;
-        }
+        goto call_finished_callback;
     }
 
     AWS_LOGF_TRACE(
```

Comment:
Seems more like info.  TRACE is all but unusable for anything other than protocol-level debugging.

Reply:
Changed all traces to debug.

Reply:
This one's going to happen a lot, so keeping this at DEBUG.